src/icons/fileIcons.ts

```diff
@@ -1367,5 +1367,6 @@ export const fileIcons: FileIcons = {
     { name: 'lolcode', fileExtensions: ['lol'] },
     { name: 'idris', fileExtensions: ['idr', 'ibc'] },
     { name: 'quasar', fileNames: ['quasar.conf.js'] },
+    { name: 'elyra-pipeline', fileExtensions: ['pipeline'] }
```

Comment:
Thank you very much, I really like the icon. As the file extension is quite general, which means that not all files possibly ending with `.pipeline` would be related to elyra, I'd suggest to simply rename the file icon to just "pipeline". The icon could also be used for other possible frameworks which are using files with the extension `.pipeline`.

So would you mind changing it as shown below:

```diff
+ { name: 'pipeline', fileExtensions: ['pipeline'] }
- { name: 'elyra-pipeline', fileExtensions: ['pipeline'] }
```

and also rename the SVG icon file to "pipeline.svg".

Reply:
done :)